candidate-standard/recommendations/core/REC_edr-geojson.adoc

```diff
@@ -0,0 +1,6 @@
+[[rec_core_edr-geojson]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:rec-id}* |*/rec/core/edr-geojson*
+^|A |If a collection using other EDR queries uses the <<rc_items-section,`items`>> query, implementations SHOULD consider support for the EDR GeoJSON Schema as an encoding.
+|===
```

Comment:
Making this a recommendation and I linked to it in the items query section. Make sense?

Reply:
@dblodgett-usgs is there an EDR GEo-JSON schema as opposed to vanilla Geo-JSON schema?